src/main/java/de/rwth/idsg/steve/utils/DateTimeUtils.java

```diff
@@ -20,88 +20,98 @@
 
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
-import org.joda.time.LocalDateTime;
-import org.joda.time.Period;
-import org.joda.time.format.DateTimeFormat;
-import org.joda.time.format.DateTimeFormatter;
-import org.joda.time.format.PeriodFormatter;
-import org.joda.time.format.PeriodFormatterBuilder;
 
-import java.util.concurrent.TimeUnit;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
 
 /**
  * @author Sevket Goekay <sevketgokay@gmail.com>
  */
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class DateTimeUtils {
 
-    private static final DateTimeFormatter HUMAN_FORMATTER = DateTimeFormat.forPattern("yyyy-MM-dd 'at' HH:mm");
-    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormat.forPattern("HH:mm");
+    private static final DateTimeFormatter HUMAN_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd 'at' HH:mm");
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
 
-    private static final PeriodFormatter PERIOD_FORMATTER = new PeriodFormatterBuilder()
-            .printZeroNever()
-            .appendDays().appendSuffix(" day", " days").appendSeparator(" ")
-            .appendHours().appendSuffix(" hour", " hours").appendSeparator(" ")
-            .appendMinutes().appendSuffix(" minute", " minutes").appendSeparator(" ")
-            .appendSeconds().appendSuffix(" second", " seconds")
-            .toFormatter();
-
-    public static DateTime toDateTime(LocalDateTime ldt) {
+    public static OffsetDateTime toOffsetDateTime(LocalDateTime ldt) {
         if (ldt == null) {
             return null;
         } else {
-            return ldt.toDateTime();
+            return ldt.atOffset(ZoneOffset.UTC);
```

Comment:
When possible, I think the offset should be taken from the location of the station

Reply:
https://github.com/steve-community/steve/issues/96